client/src/components/memeShowPage.js

```diff
@@ -0,0 +1,32 @@
+import React, {useState, useEffect} from "react"
+
+const memeShowPage = (props) => {
+    const [meme, setMeme] = useState({})
+
+    const getMeme = async() => {
+        const memeId = props.match.params.id
+        try {
+            const response = await fetch(`/api/v1/memes/${memeId}`)
+            if(!response.ok){
+                throw new Error(`${response.status} (${response.statusText})`)
+            }
+            const parsedResponse = await response.json()
+            setMeme(parsedResponse.meme)
+        } catch (error) {
+            console.log(`Error in fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getMeme()
+    }, [])
+
+    return(
+        <div>
+            <h1>{meme.title}</h1>
+            <img src={meme.memeUrl}/>
+        </div>
```

Comment:
`divs` should now no longer exist unless they are a target element that you style with a `className`. Prefer using a React Fragment instead